src/test/java/org/broadinstitute/hellbender/engine/dataflow/DataflowTestUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.testing.DataflowAssert;
+import com.google.cloud.dataflow.sdk.transforms.Create;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import com.google.common.collect.Lists;
+import org.testng.Assert;
+
+public class DataflowTestUtils {
```

Comment:
This seems generally useful for all dataflow testing. Could you move this to src/main/utils/test or somewhere like that so that it will be packaged as part of our jar.